APE0.rst

```diff
@@ -122,18 +122,22 @@ Electing the Coordination Committee
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 A Coordination Committee election is held annually to fill any vacant seats.
 Notice of the election must be posted via the Project’s standard communication
-channels (reaching all active members) at least one month before the election
-begins. The election consists of two phases, each lasting two weeks:
+channels for users, contributors and voting members, at least one month before
+the voting phase begins. The election consists of two phases, each lasting at
+least two weeks:
 
 * Phase 1, Nominations: Candidates will be nominated and confirm their interest
   in serving. Candidates can be nominated by anyone, including themselves, and
   do not have to be `Voting Members <votingmembers>`_ to be nominated.
-  Candidates should provide a brief statement in support of their candidacy.
+  Candidates should provide a brief statement in support of their candidacy,
+  these statements should be made public as they are received during the
+  nominations period.
```

Comment:
This change makes it so that nominations have to be made public as they come in, not in one lump at the end of the nominations period.

Reply:
Nominations are not currently public as they require a discourse login - it would be good to make them fully public IMHO

Reply:
I agree the should be fully public, but when I go to https://community.openastronomy.org/c/astropy/8 in an incognito window I can see the list of CoCo nominations and read them without logging in. Should we also send links to them to the astropy and/or astropy-dev lists?

Reply:
(so that in addition to being public they are easier to find)